spcs-basic/app/README.md

```diff
@@ -0,0 +1,20 @@
+## Welcome to your First Native Application!
+
+In this application, you will be able to explore some basic concepts such as application role, versioned schemas and creating procedures and functions within a setup script.
```

Comment:
This readme text in general doesn't make sense anymore. Because the docs for NA+SPCS are pending, we might not be able to link to them yet, but let's make sure this content reflects the app nonetheless.